internal/alerts/sortfilter.go

```diff
@@ -0,0 +1,47 @@
+package alerts
+
+import (
+	"context"
+	"slices"
+	"strings"
+
+	"github.com/nais/api/internal/graph/sortfilter"
+)
+
+var SortFilter = sortfilter.New[Alert, AlertOrderField, *TeamAlertsFilter]()
+
+func init() {
+	SortFilter.RegisterSort("NAME", func(ctx context.Context, a, b Alert) int {
+		return strings.Compare(a.GetName(), b.GetName())
+	}, "STATE")
```

Comment:
Litt redd for at for hvordan det går når `NAME` og `STATE` har fallback til hverandre. Kanskje registrere en `_ENVIRONMENT` også som er fallback for `NAME`? 

Kan også være jeg husker feil og at det ikke er fare for en evig sirkulær sortering